Sources/BarcodeHeroUI/Controllers/BHCameraScanController.swift

```diff
@@ -6,14 +6,50 @@
     import Foundation
     import UIKit
 
+    #warning("TODO: Make the controller work well in any orientation.")
```

Comment:
ah I guess this is why the orientations were deleted? whats the pain point for supporting these?

Reply:
Right. No big pain point, just out of scope for getting SPM working. Worth circling back to so I made it a `#warning`.